cloudmailer/requirements.txt

```diff
@@ -0,0 +1,2 @@
+configparser
```

Comment:
as far as I have seen:
- configparser is not needed as a pip package. This comes automatically with python "default" libraries, like sys
- python-keystoneclient and python-novaclient are required pip packages

Reply:
You are right, I don't know what was wrong in the environment that asked me for configparser. And I added the other two modules.